osc/conf.py

```diff
@@ -1007,6 +1007,11 @@ def get_config(override_conffile=None,
         else:
             api_host_options[apiurl]['trusted_prj'] = []
 
+        if cp.has_option(url, 'downloadurl'):
+            api_host_options[apiurl]['downloadurl'] = cp.get(url, 'downloadurl')
+        else:
+            api_host_options[apiurl]['downloadurl'] = None
+
```

Comment:
Could you add a comment here saying that the option is unstable/experimental and may change or disappear without prior notice? It might be also good to describe here what the option does.

Reply:
Changed :)